docs/apis-clients/console-api-reference.md

```diff
@@ -4,6 +4,14 @@ title: Console API clients (REST)
 description: "To interact with Camunda Platform 8 programmatically without using the Camunda Platform 8 UI, create Cloud API clients."
 ---
 
+## Console API (REST)
+
+For all requests, include the access token for Cloud API in the Authorization header: `authorization:Bearer ${TOKEN}`.
+
+:::note
+A detailed API description can be found [here](https://console.cloud.camunda.io/customer-api/openapi/docs/#/).
```

Comment:
I would call this interactive for users with a valid C8 account. Also might be good to check in with @ultraschuppi to see if this is available for users after their trial is over (and they haven't converted to a paid tier).

Reply:
I'm also logged into C8, but I'm not able to see anything render at this URL. The Favicon loads, but the page doesn't. Is there anything else we need to mention to help someone reach the content on this page?

Reply:
<img width="1368" alt="image" src="https://user-images.githubusercontent.com/1051363/176465328-b7ca98b1-c955-4287-a769-c294ccd5b418.png">

🤷‍♀️

Reply:
This is rendering for me, thanks @ultraschuppi!